front/lib/api/chat.ts

```diff
@@ -181,9 +184,18 @@ export async function storeChatSession(
               role: m.role,
               message: m.message,
               chatSessionId: chatSession.id,
+              feedback: m.feedback,
             },
             { transaction: t }
           );
+          const chatSession2: ChatSessionType = {
+            id: chatSession.id,
+            userId: chatSession.userId,
+            created: chatSession.createdAt.getTime(),
+            sId: chatSession.sId,
+            messages: messages,
+          };
+          logChatMessageFeedback(m, owner, chatSession2);
```

Comment:
It's weird to call it always even if there is no feedback no?

Reply:
Am I missing the logic here?

Reply:
No you aren't, we should indeed proceed differently